src/CodeGeneration.Roslyn.Engine/CompilationGenerator.cs

```diff
@@ -34,29 +33,48 @@ public class CompilationGenerator
         private readonly Dictionary<string, (PluginLoader loader, Assembly assembly)> cachedPlugins = new Dictionary<string, (PluginLoader, Assembly)>(StringComparer.OrdinalIgnoreCase);
```

Comment:
```suggestion
        private readonly Dictionary<string, (PluginLoader Loader, Assembly Assembly)> cachedPlugins = new Dictionary<string, (PluginLoader, Assembly)>(StringComparer.OrdinalIgnoreCase);
```
@AArnott The style cop analyzer introduced a new rule [SA1316.md](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1316.md). Should I change according the rule?

Reply:
Let's go ahead and fix them. But let's set stylecop.json to allow for PascalCasing per [configuration](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md#tuple-element-names).